wiki/Oppia-Android-Code-Coverage.md

```diff
@@ -0,0 +1,375 @@
+## Table of Contents
```

Comment:
I think it's important to mention somewhere that the team's philosophy (and implementation) of code coverage is that we only analyze a source unit within the context of its own tests (that is, we don't care if a utility has incidental code coverage because another, unrelated test references it). This actually makes code coverage percentage numbers much more valuable since they force developers to ensure a unit itself is directly and well covered via its own tests.

Reply:
Got it. Added this section to address this: https://github.com/oppia/oppia-android/blob/e97b1abe555b08c633c5300f01b7ea7dd1e1f359/wiki/Oppia-Android-Code-Coverage.md#unit-centric-coverage-philosophy, but not sure where to place it. Initially considered placing it right at the start in the 'Understanding Code Coverage' section, but also felt that including it there might confuse or complicate things for those starting out. So placed it as a separate section. Would adjust the contents and position as per feedback.

Reply:
I think the placement that you chose seems reasonable--I don't have any thoughts on it needing to be anywhere else. Thanks for adding it!